OpenRA.Mods.Common/EditorBrushes/EditorActorBrush.cs

```diff
@@ -89,8 +89,13 @@ public bool HandleMouseInput(MouseInput mi)
 
 			if (mi.Button == MouseButton.Right)
 			{
-				editorWidget.ClearBrush();
-				return true;
+				if (mi.Event == MouseInputEvent.Up)
+				{
+					editorWidget.ClearBrush();
+					return true;
+				}
+				else
```

Comment:
Style nit: you can drop the else statement and have the `return false` unconditional because the if branch returns.